api/files/serializers.py

```diff
@@ -393,6 +401,11 @@ class FileSerializer(BaseFileSerializer):
         help_text='Whether to mark the file as unviewed for the current user',
     )
 
+    cedar_metadata_records = RelationshipField(
+        related_view='files:file-cedar-metadata-records-list',
+        related_view_kwargs={'file_id': 'get_file_guid_or_id'},
```

Comment:
```suggestion
        related_view_kwargs={'file_id': '<_id>'},
```
If the related view uses `get_file`, the helper function is unnecessary.

Reply:
One issue is, this one uses the `file_id` by default but it should be `guid` as the first choice.

```
            "cedar_metadata_records": {
                "links": {
                    "related": {
                        "href": "http://localhost:8000/v2/files/65b1834bde95ed00a1f9af6a/cedar_metadata_records/",
                        "meta": {}
                    }
                }
            }
```

but I think the original design is to return

```
...
   "href": "http://localhost:8000/v2/files/zy3q9/cedar_metadata_records/",
...
```

supporting `file_id` when guid somehow is not provided is a compatibility improvement we added later

Reply:
Update, per discussion, FE shouldn't care or request how the link is constructed, as long as we provide the link